docs/license.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: doc
+---
+
+# License
+
+ImapTest is released under the [MIT License](https://github.com/dovecot/imaptest/blob/82d49b1ea5e4ce2e54f95fb0f9711a1875daa07d/COPYING.MIT).
+
+ImapTest links to [Dovecot CE libraries](https://github.com/dovecot/core/blob/382425c927c730ff037afec2da0435c5f7d81d43/COPYING), which are mainly MIT or LGPLv2.1.
```

Comment:
"main" instead of a specific commit ID?

Reply:
This was done intentionally - simply because if the link names would ever change in source, these are permalinks and would continue to work.
Highly doubt this will ever happen ... but doesn't hurt anything either.